src/cart/crt_init.c

```diff
@@ -596,40 +599,43 @@ crt_protocol_info_free(struct crt_protocol_info *protocol_info)
 int
 crt_init_opt(crt_group_id_t grpid, uint32_t flags, crt_init_options_t *opt)
 {
-	char		*provider_env;
-	char		*interface_env;
-	char		*domain_env;
-	char		*auth_key_env;
-	char		*tmp;
-	struct timeval	now;
-	unsigned int	seed;
-	const char	*path;
-	bool		server;
-	int		rc = 0;
-	char		*provider_str0 = NULL;
-	char		*provider_str1 = NULL;
-	crt_provider_t	primary_provider;
-	crt_provider_t	secondary_provider;
-	crt_provider_t	tmp_prov;
-	char		*port_str, *port0, *port1;
-	char		*iface0, *iface1, *domain0, *domain1;
-	char		*auth_key0, *auth_key1;
-	int		num_secondaries = 0;
-	bool		port_auto_adjust = false;
-	int		i;
+	char          *provider, *provider_env;
+	char          *interface, *interface_env;
+	char          *domain, *domain_env;
+	char          *auth_key, *auth_key_env;
+	struct timeval now;
+	unsigned int   seed;
+	char          *path;
+	bool           server;
+	int            rc            = 0;
+	char          *provider_str0 = NULL;
+	char          *provider_str1 = NULL;
+	crt_provider_t primary_provider;
+	crt_provider_t secondary_provider;
+	crt_provider_t tmp_prov;
+	char          *port, *port_env, *port0, *port1;
+	char          *iface0, *iface1, *domain0, *domain1;
+	char          *auth_key0, *auth_key1;
+	int            num_secondaries  = 0;
+	bool           port_auto_adjust = false;
+	int            i;
 
 	d_signal_register();
 
-	server = flags & CRT_FLAG_BIT_SERVER;
-	port_str = NULL;
-	port0 = NULL;
-	port1 = NULL;
-	iface0 = NULL;
-	iface1 = NULL;
-	domain0 = NULL;
-	domain1 = NULL;
-	auth_key0 = NULL;
-	auth_key1 = NULL;
+	provider_env  = NULL;
+	interface_env = NULL;
+	domain_env    = NULL;
+	auth_key_env  = NULL;
+	server        = flags & CRT_FLAG_BIT_SERVER;
+	port_env      = NULL;
+	port0         = NULL;
+	port1         = NULL;
+	iface0        = NULL;
+	iface1        = NULL;
+	domain0       = NULL;
+	domain1       = NULL;
+	auth_key0     = NULL;
+	auth_key1     = NULL;
```

Comment:
I can see this isn't your code but this is *weird*, can you move these initialisers to the definitions above as we do elsewhere.

Reply:
- [ ] Move initializers into declarations.

Reply:
> * [x]  Move initializers into declarations.

Fixed with commit a63fb0d2a3032fa22eca37f6052ad69fd3fe2782